canonical-websites/snapcraft.io
# First snap flow - pagination

Fixes #1783

Adds pagination links to bottom of first snap flow steps.

### QA

- ./run or https://snapcraft-io-canonical-websites-pr-1803.run.demo.haus/
- go to first snap flow
- all steps (apart from language selection) should have a pagination links on the bottom
- pagination on OS selection should link properly to chosen OS
- pagination on 'Publish' step should enable 'next' link when snap is pushed
- all pagination links between steps should work correctly (check both Mac OS and Linux paths)

<img width="1094" alt="Screenshot 2019-04-09 at 15 06 11" src="https://user-images.githubusercontent.com/83575/55803986-c0c53c80-5adb-11e9-9047-5d6509b2b96f.png">
